llama-index-packs/llama-index-packs-ollama-query-engine/llama_index/packs/ollama_query_engine/base.py

```diff
@@ -16,15 +16,17 @@ class OllamaQueryEnginePack(BaseLlamaPack):
     def __init__(
         self,
         model: str,
+        embeddings: str = "",
         base_url: str = DEFAULT_OLLAMA_BASE_URL,
         documents: List[Document] = None,
     ) -> None:
         self._model = model
         self._base_url = base_url
-
+        self._embeddings = (embeddings if embeddings != "" else model)
+        
         llm = Ollama(model=self._model, base_url=self._base_url)
-
-        embed_model = OllamaEmbedding(model_name=self._model, base_url=self._base_url)
+            
+        embed_model = OllamaEmbedding(model_name=self._embeddings, base_url=self._base_url)
```

Comment:
what does Ollama load if you pass in `""` as the default?

Reply:
Look at line 25. In case of empty string, Same model is used as both LLM and embeddings model, just like earlier.